pico_w/bt/picow_bt_example_common.c

```diff
@@ -26,15 +28,17 @@ int btstack_main(int argc, const char * argv[]);
 const btstack_audio_sink_t * btstack_audio_pico_sink_get_instance(void);
 #endif
 
+#ifdef ENABLE_BLE
 static btstack_packet_callback_registration_t hci_event_callback_registration;
-
+#endif
 static int led_state = 0;
 
 void hal_led_toggle(void){
     led_state = 1 - led_state;
     cyw43_arch_gpio_put(CYW43_WL_GPIO_LED_PIN, led_state);
 }
 
+#ifdef ENABLE_BLE
```

Comment:
It looks wrong to me to remove the packet_handler. I don't believe it's BLE specific is it?